protocol/indexer/flags.go

```diff
@@ -16,12 +16,13 @@ type IndexerFlags struct {
 
 // List of default values
 const (
-	DefaultMaxRetries = 3
+	DefaultMaxRetries = 20
 )
 
 // List of CLI flags
 const (
-	FlagKafkaConnStr         = "indexer-kafka-conn-str"
```

Comment:
Nit: I think this is a lint error?

Reply:
I think adding the comment underneath caused the linter to not add the spacing.